osh.spec

```diff
@@ -225,8 +225,7 @@ fi
 %systemd_preun osh-worker.service
 
 %postun worker
-# osh-worker does not implement reload and restart would interrupt running tasks
-%systemd_postun osh-worker.service
+%systemd_postun_with_reload osh-worker.service
```

Comment:
This does not work as expected on RHEL-9.  On a subsequent update of the `osh-worker` package I can see:
```
  Running scriptlet: osh-worker-0.9.7.20240322.150052.ga358acc7.pr_226-1.el9.noarch                                                                                                      4/6 
/var/tmp/rpm-tmp.uo5laJ: line 1: fg: no job control
warning: %postun(osh-worker-0.9.7.20240322.150052.ga358acc7.pr_226-1.el9.noarch) scriptlet failed, exit status 1

Error in POSTUN scriptlet in rpm package osh-worker
```

When I download the corresponding RPM package, I can see the `%systemd_postun_with_reload` macro unexpanded in the script:
```
% rpm -qp osh-worker-0.9.7.20240322.150052.ga358acc7.pr_226-1.el9.noarch.rpm --scripts
warning: osh-worker-0.9.7.20240322.150052.ga358acc7.pr_226-1.el9.noarch.rpm: Header V4 RSA/SHA256 Signature, key ID 00cf68b9: NOKEY
postinstall scriptlet (using /bin/sh):

 
if [ $1 -eq 1 ] && [ -x "/usr/lib/systemd/systemd-update-helper" ]; then 
    # Initial installation 
    /usr/lib/systemd/systemd-update-helper install-system-units osh-worker.service || : 
fi
preuninstall scriptlet (using /bin/sh):

 
if [ $1 -eq 0 ] && [ -x "/usr/lib/systemd/systemd-update-helper" ]; then 
    # Package removal, not upgrade 
    /usr/lib/systemd/systemd-update-helper remove-system-units osh-worker.service || : 
fi
postuninstall scriptlet (using /bin/sh):
%systemd_postun_with_reload osh-worker.service
^^^^^^^^^^^^^^^^^^^^^^^^^^^
```